app/controllers/customers/products_controller.rb

```diff
@@ -0,0 +1,11 @@
+module Customers
+  class ProductsController < ApplicationController
+    before_action :require_customer, only: [:index]
```

Comment:
jak jest tylko jedna akcja to nawiasy ([]) są zbędne

Reply:
ok